strawberry/static/graphiql.html

```diff
@@ -110,7 +110,11 @@
       const fetchURL = window.location.href;
 
       function httpUrlToWebSockeUrl(url) {
-        return url.replace(/(http)(s)?\:\/\//, "ws$2://");
+        const parsedURL = new URL(url);
+        const protocol = parsedURL.protocol === "http:" ? "ws:" : "wss:"
```

Comment:
(if someone else reviewing this is wondering: yes, the protocols returned by `parsedURL.protocol` include a colon)

Reply:
that's odd! :D